tests/test_habitat_quality.py

```diff
@@ -176,10 +176,15 @@ def setUp(self):
         # this lets us delete the workspace after its done no matter the
         # the rest result
         self.workspace_dir = tempfile.mkdtemp()
+        #dir_path = os.path.join('C:', os.sep, 'Users', 'ddenu', 'Documents',
+        #        'hq-rarity-test-changes')
+        #os.mkdir(dir_path)
+        #self.workspace_dir = dir_path
```

Comment:
Some leftover code here

Reply:
Oops! Removed in https://github.com/natcap/invest/pull/728/commits/cf5921726822d70ff12f0b95e9d6c27e7d89c821